installers/dhall/jormungandr-testnet.dhall

```diff
@@ -0,0 +1,9 @@
+{ name = "jormungandr-testnet"
+, keyPrefix = "none"
+, relays = "FIXME"
```

Comment:
todo, figure out how to pass the list of trusted peers from here?? to jormungandr